packages/api-client/src/components/AddressBar/AddressBar.vue

```diff
@@ -126,79 +124,72 @@ function updateRequestPath(url: string) {
+            isSquare
+            :method="activeRequest.method"
+            teleport
+            @change="updateRequestMethod" />
+        </div>
 
-            <!-- Path + URL + env vars -->
-            <CodeInput
-              ref="addressBarRef"
-              aria-label="Path"
-              class="outline-none"
-              disableCloseBrackets
-              :disabled="isReadOnly"
-              disableEnter
-              disableTabIndent
-              :emitOnBlur="false"
-              importCurl
-              :modelValue="activeRequest.path"
-              :placeholder="
-                activeServer?.uid &&
-                activeCollection?.servers?.includes(activeServer.uid)
-                  ? ''
-                  : 'Enter a URL or cURL command'
-              "
-              server
-              @curl="$emit('importCurl', $event)"
-              @submit="handleExecuteRequest"
-              @update:modelValue="updateRequestPath" />
-            <div class="fade-right"></div>
-          </div>
+        <!-- Servers -->
+        <AddressBarServers v-if="activeCollection?.servers?.length" />
 
-          <AddressBarHistory :open="open" />
-          <ScalarButton
-            class="relative h-auto shrink-0 overflow-hidden !pl-2 !pr-2.5 !py-1 font-bold"
-            :disabled="isRequesting"
-            @click="handleExecuteRequest">
-            <span
-              aria-hidden="true"
-              class="inline-flex gap-1 items-center">
-              <ScalarIcon
-                class="relative shrink-0 fill-current"
-                icon="Play"
-                size="xs" />
-              <span class="text-xxs lg:flex hidden">Send</span>
-            </span>
-            <span class="sr-only"> Send Request </span>
-          </ScalarButton>
+        <div
+          class="codemirror-bg-switcher scroll-timeline-x scroll-timeline-x-hidden relative flex w-full">
+          <div class="fade-left"></div>
+          <!-- Path + URL + env vars -->
+          <CodeInput
+            ref="addressBarRef"
+            aria-label="Path"
+            class="outline-none"
+            disableCloseBrackets
+            :disabled="isReadOnly"
+            disableEnter
+            disableTabIndent
+            :emitOnBlur="false"
+            importCurl
+            :modelValue="activeRequest.path"
+            :placeholder="
+              activeServer?.uid &&
+              activeCollection?.servers?.includes(activeServer.uid)
+                ? ''
+                : 'Enter a URL or cURL command'
+            "
+            server
+            @curl="$emit('importCurl', $event)"
+            @submit="handleExecuteRequest"
+            @update:modelValue="updateRequestPath" />
+          <div class="fade-right"></div>
         </div>
-      </Listbox>
+
+        <AddressBarHistory />
+        <ScalarButton
+          class="relative h-auto shrink-0 overflow-hidden !pl-2 !pr-2.5 !py-1 font-bold"
```

Comment:
```suggestion
          class="relative h-auto shrink-0 overflow-hidden pl-2 pr-2.5 py-1 font-bold"
```
we shouldn't need the importants, I believe scalarButton should be using tw-merge/cx already, right @hwkr

Reply:
I just realized this isn't part of your PR

Reply:
Yep! This is probably from before I made useBindCx

Reply:
good catch can do